matlab/finufft_isgpuarray.m

```diff
@@ -1,15 +1,22 @@
 function is_gpuarray = finufft_isgpuarray(input)
-% FINUFFT_ISGPUARRAY    check if an array is an gpuArray.
-%
-% Note: this is currently unused since GPU codes have distinct cufinufft*
-% names.
+%FINUFFT_ISGPUARRAY   Check if an array is a gpuArray.
+
+try
+    % Try calling MATLAB's built-in isgpuarray routine. If isgpuarray does
+    % not exist (e.g., because we're in Octave) then this will throw an
+    % error, which we catch below.
+    is_gpuarray = isgpuarray(input);
+catch
+    is_gpuarray = false;
+end
+
+% % check if the isgpuarray function is available
```

Comment:
Could you delete the old code - that's the only change - thanks!

Reply:
Yes, sorry! I meant to remove it.